site/_includes/macros/cards/frontmatter-card.njk

```diff
@@ -0,0 +1,56 @@
+{% from 'macros/card-authors.njk' import cardAuthors with context %}
```

Comment:
Add a comment on top of this file saying that this renders the raw FM data read from disk, not the 11ty processed version. Since I think it's probably lagely the same as `blog-card`.